yaml/_type/set.ts

```diff
@@ -5,8 +5,7 @@
 
 import type { Type } from "../_type.ts";
 
-// deno-lint-ignore no-explicit-any
-function resolveYamlSet(data: any): boolean {
+function resolveYamlSet(data: Record<PropertyKey, unknown>): boolean {
```

Comment:
```suggestion
function resolveYamlSet(data: ResultType | null): boolean {
```

Reply:
Actually, that doesn't work, because result type could also be an object or string.

Reply:
I think we should revert this to any as we don't know what types are actually passed here.

Reply:
It must be an object or `for (const key in data) {...}` would throw no?